python/IECoreMaya/NodeParameter.py

```diff
@@ -91,7 +91,7 @@ def __getattr__( self, attrName ):
 	"""
 	@staticmethod
 	def pathValidator():
-		return re.compile( "^(\|?[^\t\n\r\f\v\|]+)+\|?$" )
+		return re.compile( r"^(|?[^\t\n\r\f\v|]+)+\|?$" )
```

Comment:
Same here. There might be an extra backslash.

Reply:
addressed in 9a8db57899fce65a4a6b62be9c06aeee4de92872